### PR TITLE
ui: add ui tests for all collections

### PR DESCRIPTION
**Pull request by harunurhan**
* Adds missing ui (ss) tests for jobs, authors, literature, both search
and detail pages, on desktop and mobile screens

* Changes existing submissions ui tests to use different screen size

* INSPIR-2519

### CODE REVIEW

File: ui/ui-tests/server.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ const proxy = require('http-proxy-middleware');
...
 app.use(
   '/api',
   proxy({
-    target: 'http://host.docker.internal:5000',
+    target: 'https://qa.inspirebeta.net', // TODO: make target configurable via env variable
+    secure: false,
     onProxyReq: proxyReq => {
       proxyReq.removeHeader('Host');
-      proxyReq.setHeader('Host', 'localhost:5000');
+      proxyReq.setHeader('Host', 'qa.inspirebeta.net');
```
Comment (drjova) [inline ui/ui-tests/server.js]: left over?

File: ui/ui-tests/server.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ const proxy = require('http-proxy-middleware');
 
 const app = express();
 
-// proxy api calls to inspire-next running locally (localhost:5000)
 // necessary for recording api responses when running tests locally (for the first time or ondemand)
 app.use(
   '/api',
   proxy({
-    target: 'http://host.docker.internal:5000',
+    target: 'https://qa.inspirebeta.net', // TODO: make target configurable via env variable
```
Comment (drjova) [inline ui/ui-tests/server.js]: ditto

File: ui/ui-tests/tests/authors/detail.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    await page.goto(routes.public.authorDetail983328, {
+      waitUntil: 'networkidle0',
+    });
+    await page.waitFor(selectors.searchResults);
+
+    const desktopSS = await takeScreenShotForDesktop(page);
+    // FIXME: citations by year graph is rendered incorrectly sometimes, therefore `failureThreshold`
+    expect(desktopSS).toMatchImageSnapshot({
+      failureThreshold: '0.071',
```
Comment (drjova) [inline ui/ui-tests/tests/authors/detail.test.js]: :(

Comment (harunurhan) [reply ui/ui-tests/tests/authors/detail.test.js]: we talked about this... it's an unfortunate line of code for all of us 😢

Comment (drjova) [reply ui/ui-tests/tests/authors/detail.test.js]: I just wanted to share my sadness about this

**Pull request merged**